cmd/limactl/prune.go

```diff
@@ -16,15 +22,97 @@ func newPruneCommand() *cobra.Command {
 		RunE:              pruneAction,
 		ValidArgsFunction: cobra.NoFileCompletions,
 	}
+
+	pruneCommand.Flags().BoolP("fallback", "F", false, "Only prune images without a digest specified (fallback images usually)")
+	pruneCommand.Flags().BoolP("unreferenced", "U", false, "Only prune downloads not referenced in any VM or template")
```

Comment:
```suggestion
	pruneCommand.Flags().Bool("unreferenced-only", false, "Only prune downloads not referenced in any VM")
```
Blobs referred by templates are not worth keeping

Reply:
That was my opinion too.. but there are ppl who wants this: https://github.com/lima-vm/lima/issues/1410#issuecomment-1461496682

Reply:
Why would you not want to keep images referenced in templates. You will only have images in the cache that you have used before, so it is not unreasonable that you want to use them again.

If you stop using a specific template, eventually the template will be updated to refer to a newer version of the image, and the old cached version will then become unreferenced.

So just because I currently don't have a fedora instance doesn't mean I want to prune the current fedora image. I only want to prune the old versions of fedora that are not being used by templates anymore, as it is now unlikely that I will ever use them again.

Reply:
As I understand there is no easy way to calculate the age of an image w/o taking special actions..

Reply:
Sorry, I meant something other actually.. it's easy to get the age of an image for sure.. but I'd say users would like to keep recenlty *used* images

Reply:
> to keep recenlty used images..

`st_atime` isn't enough?

Reply:
Well.. on MacOS it might work but it's almost always disabled on Linux systems. Not sure how many Linux users we have 🤔

Reply:
@AkihiroSuda @jandubois How about removing the template references part and merge this PR after? We can discuss another improvements separately then.

Reply:
Downloads referenced by a template are dropped from the PR scope.